### ISSUE TITLE
newline seems to get removed...

### ISSUE THREAD

**Issue by ckressibucher** [mihaeu/html-formatter#1]
line feeds inside text nodes are silently removed...
I worked around this by replacing line feeds with other whitespace before (which is probably not the most elegant solution):

```
            $content = str_replace("\r\n", "\n", $content);
            $content = str_replace("\n", " ", $content);
            $content = HtmlFormatter::format($content);
```

**Comment by mihaeu** [comment]
Thanks Claudio, I'll have a look at it tomorrow.

**Comment by mihaeu** [comment]
Sorry for taking so long @ckressibucher I kind of forgot about it over the weekend :(

Anyways, I used this as an excuse to clean up the code a bit and made it compliant with the recent PSRs. I also started adding some tests to confirm the bug you found.

Now the question is how to go about solving it. Removing the line feeds won't change the output of the html and will make it look nice when line wrapping is turned off. If line wrapping is on then the line breaks would also have to be indented.

So this

``` html
<p>Test
    and some more
    Linefeeds</p>
```

would look something like (with space indents after the line feed)

``` html
<p>
    Test
    and some more
    Linefeeds
</p>
```

and the way it is implemented now would produce:

``` html
<p>
    Test some more Linefeeds
</p>
```

Final output in the browser would be the same.

Suggestions?
